src/main/java/net/snowflake/client/core/ColumnTypeHelper.java

```diff
@@ -0,0 +1,23 @@
+package net.snowflake.client.core;
```

Comment:
the copyright header is missing - check also other new files

Reply:
Fixed